app/components/Edit/EditSections.js

```diff
@@ -60,6 +60,7 @@ class EditSections extends React.Component {
 	    let url = '/api/resources/' + resourceID;
 	    fetch(url).then(r => r.json())
 	    .then(data => {
+            data.resource.services[0].inherit_schedule = true;
```

Comment:
Maybe break this out into two lines so it is more readable? Create a variable that is descriptive so that it isn't just you assigning some property to a random object. Something like the following:
let scheduleService = data.resource.services[0];
scheduleService.inherit_schedule = true;